packages/server/email/notificationSummaryCreator.tsx

```diff
@@ -27,19 +29,39 @@ ${ExternalLinks.TEAM}
 `
 }
 
-const notificationSummaryCreator = (props: NotificationSummaryProps) => {
+interface Props {
+  appOrigin: string
+  preferredName: string
+  notificationCount: number
+  environment: ServerEnvironment
+}
+
+const notificationSummaryCreator = async (props: Props) => {
   const {notificationCount} = props
   const subject = `You have ${notificationCount} new ${plural(
     notificationCount,
     'notification'
   )} 👀`
+
+  const NotificationSummaryEmailRoot =
+    require('parabol-client/modules/email/components/NotificationSummaryEmailRoot').default
+
+  const bodyContent = await renderSSRElement(
+    <NotificationSummaryEmailRoot {...props} environment={props.environment as any} />,
+    props.environment
+  )
+
+  const html = emailTemplate({
+    bodyContent,
+    title: subject,
+    previewText: subject,
+    bgColor: PALETTE.SLATE_200
+  })
+
   return {
     subject,
     body: textOnlySummary(props),
-    html: Oy.renderTemplate(<NotificationSummaryEmail {...props} />, {
-      title: subject,
-      previewText: subject
-    })
```

Comment:
out of curiosity, how different is using `0y` to render vs the template you used? did you have any particular reasons to drop the `0y` usage here?

Reply:
As I understand, our usage of `Oy` predates the existence of `emailTemplate`, and the latter is preferred (a few emails were switched from `Oy` to `emailTemplate` in https://github.com/ParabolInc/parabol/pull/2881), though @mattkrick may have better context, including whether this is a correct change.

Reply:
IIRC oy is just guardrails to make sure you use components that will render correctly in email clients. if we can do away with it, that's awesome!